# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/labplot/hotfix-qttimer.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

labplot-2.12.1/src/backend/worksheet/plots/cartesian/XYFourierFilterCurve.cpp:97:9: error: ‘timer’ was not declared in this scope; did you mean ‘timex’?

--- labplot-2.12.1/src/backend/worksheet/plots/cartesian/XYFourierFilterCurve.cpp.vanilla	2026-02-08 18:33:42.112806649 +0100
+++ labplot-2.12.1/src/backend/worksheet/plots/cartesian/XYFourierFilterCurve.cpp	2026-02-08 18:34:02.963179828 +0100
@@ -27,6 +27,7 @@
 #include <QDebug> // qWarning()
 #include <QIcon>
 #include <QThreadPool>
+#include <QElapsedTimer>
 
 /*!
  * \class XYFourierFilterCurve
